### ISSUE TITLE
Unable to train

### ISSUE THREAD

**Issue by Zumbalamambo** [yshean/abnormal-spatiotemporal-ae#3]
Can you please tell me how do I train with avenue dataset without docker? Im not at all able to train

**Comment by razmik** [comment]
I was able to train without docker. 
One issue was there, however [fixed by this answer.](https://stackoverflow.com/questions/42969779/keras-error-you-must-feed-a-value-for-placeholder-tensor-bidirectional-1-keras)

**Comment by LekshmyHari** [comment]
Hi @razmik, did you perform testing? Can you share details about the accuracy and the threshold values?

**Comment by razmik** [comment]
Not enough machine power to complete the training, hence could complete the training. Will share the details if I was able to in another env.

**Comment by razmik** [comment]
@Aswanykh  You can use [this code snippet](https://github.com/razmik/quick-and-dirty/blob/master/convert_tif_to_frame.py) to convert the tiff files to jpeg.

**Comment by razmik** [comment]
@LekshmyHari @Aswanykh Were you able to compose or find the groundtruth files for respective datasets?

**Comment by whrenstone** [comment]
@yshean  could you share with us as how to generate the groundtruth files. I do checked the google drive link you provided, but only get a suspicious gt_ped1.mat. Many Thanks
